scenarioo-client/app/components/detailarea/detail-accordion/detail-accordion.component.css

```diff
@@ -1,7 +1,40 @@
-ul {
-    margin-bottom: 0px;
+.panel-heading {
+    cursor: pointer;
 }
 
-sc-detail-accordion a {
-    cursor: pointer;
+span.accordionHeader {
+    margin-left: 10px;
+
+}
+
+.panel-title {
+    color: #337AB7;
+}
+
+accordion {
+    margin-right: 20px;
+    margin-left: 10px;
+}
+
+accordion accordion-group {
```

Comment:
the important selector should be avoided if possible. Usually it's only needed if a selector is not specific enough https://uxengineer.com/css-specificity-avoid-important-css/